centreon/www/class/centreonGraph.class.php

```diff
@@ -196,8 +196,11 @@ public function __construct($user_id, $index = null, $debug = 0, $compress = nul
         $this->onecurve = false;
         $this->checkcurve = false;
 
-        $DBRESULT = $this->DBC->query("SELECT RRDdatabase_path, RRDdatabase_status_path FROM config LIMIT 1");
-        $config = $DBRESULT->fetch();
+
+        $DBRESULT = $this->DBC->prepare("SELECT RRDdatabase_path, RRDdatabase_status_path FROM config LIMIT 1");
```

Comment:
There is no need for a prepare() if you don't bind any values, a query() is enought here